dbt/include/databricks/macros/materializations/table.sql

```diff
@@ -29,8 +29,9 @@
 
   {% set should_revoke = should_revoke(old_relation, full_refresh_mode=True) %}
   {% do apply_grants(target_relation, grant_config, should_revoke) %}
-  {% do apply_tblproperties_python(target_relation, tblproperties, language) %}
-
+  {% if language=="python" %}
+    {% do apply_tblproperties(target_relation, tblproperties) %}
+  {% endif %}
   {%- do apply_tags(target_relation, tags) -%}
 
   {% do persist_docs(target_relation, model, for_relation=language=='python') %}
```

Comment:
@benc-db sorry for going back into a closed PR - but I'm curious, why only do persist_docs if the language is 'python' and the materialisation is table?

Reply:
To more clear about the "why" behind my question: we're materialising a model as a table in Databricks and I've been trying to understand why the persist_docs behaviour isn't yielding what I'd expect (i.e. persisted documentation).

Reply:
for_relation flag says apply a model-level comment.  For SQL models, we apply the model-level comment in the table create SQL, so we only need to persist it after table create for python models.

Reply:
Thank you for clarifying, really appreciate it.